Gruntfile.js

```diff
@@ -135,11 +135,10 @@ grunt.initConfig({
 		}
 	},
 
-	// Remove the requireSpacesInsideParentheses override once everything is fixed
 	jscs: {
 		all: {
 			options: {
-				requireCapitalizedComments: null
+				config: ".jscsrc"
```

Comment:
For the config file yes true should be fine. The bug you mentioned with max errors was already reported to them https://github.com/jscs-dev/node-jscs/issues/1722 it is unrelated to this as its only related to running with fix. I added it here just because the normal max is 50 and i believe we want to output them all.